garage/tf/core/networks.py

```diff
@@ -0,0 +1,48 @@
+import tensorflow as tf
+
+
+def mlp(input_var,
```

Comment:
There's been some discussion towards using the `tf.keras` namespace instead of `tf.contrib`, but I don't think it matters in this case.